neurom/fst/_neuronfunc.py

```diff
@@ -189,16 +191,13 @@ def sholl_frequency(nrn, neurite_type=NeuriteType.all, step_size=10):
     nrns = neuron_population(nrn)
     neurite_filter = is_type(neurite_type)
 
-    min_soma_edge = float('Inf')
-    max_radii = 0
-    neurites_list = []
-    for neuron in nrns:
-        neurites_list.extend(((neurites, neuron.soma.center)
-                              for neurites in neuron.neurites
-                              if neurite_filter(neurites)))
+    neurites_list = [(neurites, neuron.soma.center)
+                     for neuron in nrns
+                     for neurites in neuron.neurites
+                     if neurite_filter(neurites)]
 
-        min_soma_edge = min(min_soma_edge, neuron.soma.radius)
```

Comment:
this is only getting computed once per neuron; now we're traversing nrns multiple times, no?

Reply:
Wow, I must have been tired when I wrote this code. I thought it was being computed for every neurite. Then I am not sure of what needs to be fixed now...